api/src/main/java/org/pmiops/workbench/conceptset/ConceptSetService.java

```diff
@@ -227,6 +211,17 @@ public DbConceptSet cloneConceptSetAndConceptIds(
     return conceptSetDao.findByWorkspaceId(workspace.getWorkspaceId());
   }
 
+  private DbConceptSet getDbConceptSet(Long conceptSetId, Long workspaceId) {
```

Comment:
nit: for all of these methods my preference would be to reverse the parameter order to be symmetric with the API / REST pattern, i.e. workspaceId, conceptSetId

Reply:
Done.